### ISSUE TITLE
Incorrect typings for `set`

### ISSUE THREAD

**Issue by WaltzingPenguin** [upstash/upstash-redis#134]
```js
const redis = new Redis({
  url: MY_URL,
  token: MY_TOKEN
})
const data = redis.set('key', 'value')
```

`redis.set` erroneously has the type of `<TData>(key: string, value: TData, opts?: SetCommandOptions | undefined) => Promise<TData>`

The return value should be `Promise<TData | 'OK' | null>`

**Comment by chronark** [comment]
👍 thank you. I'll fix it.

**Comment by chronark** [comment]
fixed it in [v1.11.0](https://github.com/upstash/upstash-redis/releases/tag/v1.11.0)
